modules/shared/components/molecules/TextPollViewOptions/__snapshots__/TextPollViewOptions.test.tsx.snap

```diff
@@ -2,10 +2,9 @@
 
 exports[`OptionGroup component should render () OptionGroup when we apply () 1`] = `
 <div
-  className="mb-m"
```

Comment:
Margin 16px between TextPollViewOptions and the footer should be applied.

Reply:
that's what space-y-4 do instead of giving each element a margin